src/environment.js

```diff
@@ -1,23 +1,52 @@
 // @flow
-import { Environment, Network, RecordSource, Store } from 'relay-runtime'
+import 'regenerator-runtime/runtime'
+import { Environment, RecordSource, Store } from 'relay-runtime'
+import type { Middleware } from 'react-relay-network-modern/lib/definition'
+import uuid from 'uuid/v4'
+import { RelayNetworkLayer, RelayNetworkLayerRequestBatch } from 'react-relay-network-modern'
 
 const store: * = new Store(new RecordSource())
 
-const network: * = Network.create((operation: *, variables: *): * =>
-    // TODO: fallback required here
-    // eslint-disable-next-line
-    fetch('http://localhost:9090/api', {
-        body: JSON.stringify({
-            query: operation.text,
-            variables
-        }),
-        headers: {
-            Accept: 'application/json',
-            'Content-Type': 'application/json'
-        },
-        method: 'POST'
-    }).then((response: *): * => response.json())
-)
+type ReplacedFields = {
+    files: { [id: string]: File | Blob },
+    variables: { [id: string]: mixed },
+}
+
+const recReplace = (variables: Object): ReplacedFields => Object.entries(variables).reduce((acc: ReplacedFields, [key, value]: *): * => {
```

Comment:
This is awesome BTW... What does `acc` mean though?

Reply:
`accumulator` just the default variable in `Array::reduce`